ci-operator/config/openshift/crd-schema-gen/OWNERS

```diff
@@ -0,0 +1,8 @@
+reviewers:
+  - sttts
+  - damemi
+  - soltysh
+approvers:
+  - sttts
+  - damemi
+  - soltysh
```

Comment:
nit: missing '\n'

Reply:
preferably if you could add me as well because I have a feeling that I am gonna be the first person fixing CI :)